tests/cacl/test_control_plane_acl.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import time
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer  # disable automatic loganalyzer globally
+]
+
+SONIC_SSH_PORT  = 22
+SONIC_SSH_REGEX = 'OpenSSH_[\\w\\.]+ Debian'
+
+
+def test_control_plane_acls(duthost, testbed_devices, creds):
+    """Test control plane ACL functionality on a SONiC device
+    """
+    localhost = testbed_devices['localhost']
```

Comment:
testbed_devices has been deprecated.

Reply:
Thanks. Using new 'localhost' fixture.

Reply:
@jleveque You added `localhost` but forgot to remove `testbed_devices`. Test will fail with 
```
______________________________________________________________________ ERROR at setup of test_control_plane_acls _______________________________________________________________________
file /home/msosyak/bfn-sonic-mgmt/tests/cacl/test_control_plane_acl.py, line 11
  def test_control_plane_acls(duthost, localhost, testbed_devices, creds):
E       fixture 'testbed_devices' not found
```

Reply:
@msosyak: Thanks for the heads-up! PR to remove here: https://github.com/Azure/sonic-mgmt/pull/1689/files